roles/docker/defaults/main.yml

```diff
@@ -3,10 +3,7 @@ docker_apt_repo: https://download.docker.com/linux
 debian_apt_repo: "{{ docker_apt_repo }}/debian"
 ubuntu_apt_repo: "{{ docker_apt_repo }}/ubuntu"
 
-docker_debian_ce_version: "18.03.1"
-docker_ubuntu_ce_version: "18.03.1"
-docker_centos_ce_version: "18.06.0"
 docker_ce_pkg_name: docker-ce
-debian_docker_ce_version: "{{ docker_debian_ce_version }}~ce-0~debian"
-ubuntu_docker_ce_version: "{{ docker_ubuntu_ce_version }}~ce~3-0~ubuntu"
-centos_docker_ce_version: "{{ docker_centos_ce_version }}.ce"
+debian_docker_ce_version: "{{ (ansible_distribution_release == 'buster') | ternary('5:20.10.7~3-0~debian-buster', '5:19.03.15~3-0~debian-stretch') }}"
+ubuntu_docker_ce_version: "5:20.10.7~3-0~ubuntu-bionic"
```

Comment:
I haven't checked these ubuntu and debian versions yet. Are they LTS? Is there any security vulnerabilities in our current install? Update Docker needs more testing and research based on my experience, so I'd recommend creating another ticket to take care of the Docker update or the scope of this ticket will be super huge.

Reply:
There are many vulnerabilities and at least one serious vuln. I reversed this commit and I'll make a ticket to do further testing.